src/go/MONIT/alert.go

```diff
@@ -531,12 +534,36 @@ func run() {
 	}
 }
 
+//helper function for parsing Configs
+func openConfigFile(configFilePath string) {
+	jsonFile, e := os.Open(configFilePath)
+	if e != nil {
+		if configJSON.Verbose > 0 {
+			log.Printf("Config File not found at %s, error: %s", configFilePath, e)
+		} else {
+			fmt.Printf("Config File Missing at %s. Using Defaults\n", configFilePath)
+		}
+		return
+	}
+	defer jsonFile.Close()
+	decoder := json.NewDecoder(jsonFile)
+	err := decoder.Decode(&configJSON)
+	if err != nil {
+		log.Printf("Config JSON File can't be loaded, error: %s", err)
+		return
+	} else if configJSON.Verbose > 0 {
+		log.Printf("Load config from %s\n", configFilePath)
+	}
+}
+
+//function for parsing Configs
 func parseConfig(verbose int) {
 
-	configFile = os.Getenv("CONFIG_PATH") //CONFIG_PATH Environment Variable storing config filepath.
+	configFilePath = os.Getenv("CONFIG_PATH") //CONFIG_PATH Environment Variable storing config filepath.
+	defaultConfigFilePath := os.Getenv("HOME") + "/alert.json"
 
 	//Defaults in case no config file is provided
-	configJSON.CMSMONURL = "https://cms-monitoring.cern.ch"
+	configJSON.CMSMONURL = "http://cms-monitoring.cern.ch"
```

Comment:
leave the `https` (eventually we'll use it

Reply:
Fixed